cegs_portal/search/admin.py

```diff
@@ -1,8 +1,14 @@
+from django import forms
 from django.contrib import admin
+from django.db import models
```

Comment:
Remove unused imports

Reply:
These unused imports are still there.

Reply:
Imports plural? Looks like I can remove "from django.contrib import admin". May need the rest.

Reply:
Neither of these are used:
```python
from django.db import models
from django.forms import Textarea
```

Reply:
Yes, you are right. Thought I needed it for using textarea in the code but forms seem to handle that.